scene/animation/tween.cpp

```diff
@@ -1675,50 +1658,48 @@ bool Tween::targeting_property(Object *p_object, NodePath p_property, Object *p_
 	data.delay = p_delay;
 
 	// Ensure there is a valid delta
-	if (!_calc_delta_val(data.initial_val, data.final_val, data.delta_val))
-		return false;
+	ERR_FAIL_COND(!_calc_delta_val(data.initial_val, data.final_val, data.delta_val));
```

Comment:
It may now return an error when it didn't previously. Might be fine, I'm not familiar enough to say if this case is a normal silent skip or an error.

Reply:
Hmm.. Well, I returned it back, I'm not sure if this is right too )